packages/models/src/migrations/20230406150645-create-activity.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('activity', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      valid: {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true,
+      },
+      points: {
+        type: Sequelize.INTEGER,
+      },
+      currency: {
+        type: Sequelize.BLOB,
+      },
+      price: {
+        type: Sequelize.DECIMAL,
+      },
+      priceUSD: {
+        type: Sequelize.FLOAT,
+        field: 'price_usd',
+      },
+      type: {
+        type: Sequelize.STRING,
+      },
+      contractAddress: {
+        type: Sequelize.BLOB,
```

Comment:
What is the advantage of using blob vs string for addresses? One downside I can think of is that when we join the data in metabase with tables from other DBs in metabase that use string for wallet address type, we'll need to cast block -> string.

Reply:
Trying something new.  It's another method of normalization that doesn't involve string casing, since it's just a binary number.  Similar to how dune does hex strings.

Reply:
For analytics or troubleshooting purposes, given the wallet address of a user in hexa, how do you cast to a blob when writing a SQL query? I'm worried this encoding will make the data harder to use/interpret.

Reply:
```sql
SELECT encode(decode('fc874a66afe868e9fadaf5f98a4268da607b5d38', 'hex'), 'hex');
```